services/SSMConnection.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  SSMClient,
+  StartSessionCommand,
+  TerminateSessionCommand,
+} from '@aws-sdk/client-ssm';
+
+export default class SSMConnection {
+  private client: SSMClient;
+
+  constructor() {
+    this.client = new SSMClient();
+  }
+
+  async connect(
+    target: string,
+    dbHost: string,
+    dbPort: string | number,
+    localPort: string | number
+  ) {
+    const input = {
+      Target: target,
+      DocumentName: 'AWS-StartPortForwardingSessionToRemoteHost',
+      Parameters: {
+        host: [`${dbHost}`],
+        portNumber: [`${dbPort}`],
+        localPortNumber: [`${localPort}`],
```

Comment:
```suggestion
        host: [dbHost],
        portNumber: [dbPort],
        localPortNumber: [localPort],
```

Seria legal colocar a variável sem template string aq nesse carinha